app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.test.tsx

```diff
@@ -108,190 +108,222 @@ const servers: JenkinsServer[] = [
 	}
 ];
 
-describe('addConnectedState', () => {
-	it('should handle a single server', () => {
-		const singleServer: JenkinsServer[] = [servers[0]];
-		const result = addConnectedState(singleServer);
-
-		expect(result[0].connectedState).toEqual(ConnectedState.CONNECTED);
+describe('Connection Panel Suite', () => {
```

Comment:
Shuffled some existing tests around to make the describe blocks make a little more sense.